tests/letstest/scripts/test_sdists.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh -xe
 
 cd letsencrypt
-./certbot-auto --install-only -n --debug
+letsencrypt-auto-source/letsencrypt-auto --install-only -n --debug
```

Comment:
Why using the local newest letsencrypt-auto here instead of the released certbot-auto script? Does it mean the previous logic was wrong?

Reply:
We need to use letsencrypt-auto in letsencrypt-auto-source here because otherwise we fail to bootstrap and create a venv so we know whether to use Python 2 or Python 3 in this test script. We do the same kind of thing in `test_apache2.sh` and `test_test.sh`.

I wouldn't say the previous logic was necessarily wrong, just that it didn't allow the test to run on systems where bootstrapping with the latest released certbot-auto didn't work.

Reply:
OK it makes sense. And the new approach is more consistent, since the current letsencrypt-auto script becomes a part of the test, while we now that the released one is already working.